src/core/ui.js

```diff
@@ -131,6 +138,31 @@ function createWarnButton(butName, arr, title) {
   return button;
 }
 
+menu.addEventListener("keydown", e => {
+  if (e.key === "Escape") {
+    e.preventDefault();
+    menu.classList.add("respec-hidden");
+    menu.classList.remove("respec-visible");
+    respecPill.setAttribute("aria-expanded", String(menu.hidden));
+    menu.hidden = true;
+    respecPill.focus();
+  } else if (
+    e.target == menu.querySelector("li:first-child").firstElementChild &&
+    e.shiftKey &&
+    e.key === "Tab"
```

Comment:
This tab key check is repeated, maybe we can do `if (tab) { if (e.target ...) ... else if ... }` for readability?